### DISCUSSION TITLE
[Help]: Change slide size after click

### DISCUSSION THREAD

**Discussion by coldnanee** [davidjerleke/embla-carousel#734] [Help] [answered]
### Summary

Hi! I wanna to create this slider:

![Screenshot from 2024-02-04 14-02-02](https://github.com/davidjerleke/embla-carousel/assets/145275662/1c4ae4f1-c02b-44d8-aeb8-83d2e636dd30)

I add default slide styles: flex: 0 0 100%, if slide unactive : flex: 0 0 20%, if active: flex: 0 0 40%;

this is my slider options:
```
{
	loop: true,
	containScroll: 'keepSnaps',
	skipSnaps: false,
	align: 'start',
	watchDrag: false
}
```

and I got strange problem: when I am click on slide (making it active), loop and align options become disabled and slider starting to behave strangely :c

what am I doing wrong?

### If applicable, which variants of Embla Carousel are relevant to this question?

- [X] embla-carousel **(Core)**
- [X] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by davidjerleke** [comment]
@coldnanee thanks for your question. Please create a minimal CodeSandbox that reproduces the problem. I can't debug fragments of code and sccreenshots. Thanks!

**Reply by davidjerleke** [reply]
@coldnanee do you want to pursue this?

**Reply by coldnanee** [reply, marked as answer]
@davidjerleke, no, I solved my problem differently. thx for your time

**Reply by linkb15** [reply]
@coldnanee How do you solve it?

I did have similar use case here, but the animation on the last card seems glitching, like it expand first then scroll to focus, I am not able to figure how to scroll at the same time it expanded to make it smoother animation
